packages/appium/lib/extension/manifest.js

```diff
@@ -381,7 +415,7 @@ export class Manifest {
         shouldWrite = (await this.syncWithInstalledExtensions()) || shouldWrite;
       }
 
-      if (isNewFile || shouldWrite) {
+      if (shouldWrite || isNewFile) {
```

Comment:
what is the purpose of changing the condition?

Reply:
I think this may have been left over from something I reverted.  It doesn't really matter afaict, so I can revert this.